src/views/user/ProfileUserView.jsx

```diff
@@ -88,13 +87,13 @@ class MyProfile extends Component {
     }
 
     saveUsername() {
-        const newusername = this.state.NewUsername;
-        console.log(this.checkAvailibility(newusername))
-        if (this.checkAvailibility(newusername)) {
+        const newUserName = this.state.NewUsername;
+        console.log(this.checkAvailability(newUserName));
```

Comment:
Another console log that we might delete.

Reply:
Removed it